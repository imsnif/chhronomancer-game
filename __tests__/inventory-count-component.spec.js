import React from 'react'
import 'react-native'
import renderer from 'react-test-renderer'

test('Inventory-Count component => renders correctly', () => {
  const InventoryCount = require('../components/inventory-count').default
  const tree = renderer.create(
    <InventoryCount count={4} />
  ).toJSON()
  expect(tree).toMatchSnapshot()
})
