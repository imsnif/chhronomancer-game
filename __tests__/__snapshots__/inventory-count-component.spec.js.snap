// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Inventory-Count component => renders correctly 1`] = `
<View
  style={
    Object {
      "flex": 1,
      "flexDirection": "row",
    }
  }
>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#329932",
        "flex": 1,
        "flexDirection": "row",
        "fontFamily": "telegrama_raw",
        "fontSize": 30.607554623001164,
        "textAlignVertical": "center",
      }
    }
  >
    Cap: 
  </Text>
  <Text
    accessible={true}
    allowFontScaling={true}
    ellipsizeMode="tail"
    style={
      Object {
        "color": "#329932",
        "flex": 1,
        "flexDirection": "row",
        "fontFamily": "telegrama_raw",
        "fontSize": 30.607554623001164,
        "textAlignVertical": "center",
      }
    }
  >
    4
    /8
  </Text>
</View>
`;
